doc/manual/embedding.rst

```diff
@@ -33,9 +33,9 @@ We start with a simple C program that initializes Julia and calls some Julia cod
       return 0;
   }
 
-In order to build this program you have to put the path to the Julia header into the include path and link against ``libjulia``. For instance, when Julia is installed to ``$JULIA_DIR``, one can compile the above test program ``test.c`` with gcc using::
+In order to build this program you have to put the path to the Julia header into the include path and link against ``libjulia``. For instance, when Julia is installed to ``$JULIA_PREFIX/bin`` (where you are responsible for defining the ``JULIA_PREFIX`` variable appropriately) , its header file ``julia.h`` is installed in ``$JULIA_PREFIX/include`` and its library ``libjulia`` is installed in ``$JULIA_PREFIX/lib`` (by default). In this case, one can compile the above ``test.c`` program with gcc using::
```

Comment:
you have an extra space before a `,`.   There should also be a comma after "In order to build this program".

While I am nitpicking, probably the paragraph should be consistent about "you" vs. "one" — pick one and use it consistently.